src/components/reusables/Input.tsx

```diff
@@ -22,6 +23,8 @@ export default class Input extends React.Component<InputProps, {}> {
           type={this.props.type || "text"}
           name={this.props.name}
           id={this.props.name}
+          defaultValue={this.props.value}
+          readOnly={false}
```

Comment:
Hmm is this because of the `defaultValue` prop? I run into issues with the `defaultValue` and `value` props for `input` in React.

Reply:
Yes, yes it is :(